utils/mux_gen.py

```diff
@@ -72,7 +72,7 @@
 )
 
 parser.add_argument(
-    '--name-output',
+    '--name-outputs',
```

Comment:
Why do you need `--name-outputs` -- IE there should only be one output on a mux right?

Reply:
For a minute I wanted a 2 output MUXCY, I'll revert this.